web/components/feed/feed-answer-comment-group.tsx

```diff
@@ -70,17 +70,23 @@ export function FeedAnswerCommentGroup(props: {
 
   const scrollAndOpenReplyInput = useEvent(
     (comment?: Comment, answer?: Answer) => {
-      setReplyToUsername(comment?.userUsername ?? answer?.username ?? '')
+      setReplyToUser(
```

Comment:
I think this could be refactored to look a little nicer. Maybe with if statements for comment vs answer?